Formula/helmfile.rb

```diff
@@ -1,8 +1,9 @@
 class Helmfile < Formula
   desc "Deploy Kubernetes Helm Charts"
   homepage "https://github.com/roboll/helmfile"
-  url "https://github.com/roboll/helmfile/archive/v0.81.0.tar.gz"
-  sha256 "6aba51fed66930fe7445570116daa9cdd7fb3a0fa73b4758f490efc6561e8e07"
+  url "https://github.com/roboll/helmfile.git",
```

Comment:
We still prefer tarballs over git-checkouts, unless the underlying build procedure requires a git working copy (i.e. retrieving the commit refspec).

This also applies to #43273.

Reply:
ok, I will switch back then.